bitstamp/client.py

```diff
@@ -57,12 +57,11 @@ def _construct_url(self, url, base, quote):
         """
         Adds the orderbook to the url if base and quote are specified.
         """
-        #This condition allows for both None and False values
-        if not base and not quote:
+        if (base is None) and (quote is None):
```

Comment:
Thanks for the remark. There was an error in _construct_url and I wanted to make sure it was not that. I have restored the original.